config/posts.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Posts Config
+ */
+
+return [
+    // Max number of posts that can be fetched for privileged/unprivileged requests
+    'privileged_request_limit' => 100,
```

Comment:
Could we make these accessible with environment variables? i.e. `API_POST_LIST_MAX_LIMIT` and `API_POST_LIST_ADMIN_MAX_LIMIT` ?

Reply:
sure

Reply:
@tuxpiper env vars added